spec/debugs_bunny/internal/schema/index_descriptor_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+require 'debugs_bunny/internal/schema/index_descriptor'
+
+RSpec.describe DebugsBunny::Internal::Schema::IndexDescriptor do
+  let(:index_descriptor) { described_class.new(:unique_index, %w[name type], unique: true) }
```

Comment:
Being fussy here, but I think the name of the index being `:unique_index` might be confusing, since it looks like rails keywords, and not an assigned name. Even something like `:my_unique_index` could make it more obvious that it's defining a name, and is not a keyword.

Reply:
Will change